data_structures/priority_q.ts

```diff
@@ -0,0 +1,43 @@
+const minPrioryty = 10;
+const maxPriority = 0;
+
+const priorityQ = [
```

Comment:
@dianalemen, in the task you need to implement priority queue data structure and task runner.
A priority queue is described here https://www.geeksforgeeks.org/priority-queue-set-1-introduction/.

Please, read and fix your pull request

Reply:
@AlexHladin That was what I ask you about on the call. You said it's just array. 

OK. I will fix

Reply:
@AlexHladin , since, there is a tip in the article you've shared  - _**How to implement priority queue?
Using Array: A simple implementation is to use array of following structure.**_ 
The array implementation is suitable.  That is why I've only changed the structure of the task node and have added the `insert` method.

Also, I've added class implementation, just to cover it.

The result is exactly the same, you can check the attached screenshots.